pkg/openslo/v1/alert_condition_test.go

```diff
@@ -0,0 +1,171 @@
+package v1
```

Comment:
What is your take on adding `t.Parallel()` for this tests?

Reply:
sure, we can add them, although we're not making any intensive operations here, the tests are and will be VERY fast on their own, adding parallelism here has an underlying synchronisation cost too, I'd defer with using it until we have a full coverage, we can easily compare the difference then :)

Reply:
Fine, sounds like plan